_extensions/acronyms/acronyms_pandoc.lua

```diff
@@ -133,14 +133,13 @@ end
 function AcronymsPandoc.generateDefinitionList(sorted_acronyms)
     local definition_list = {}
     for _, acronym in ipairs(sorted_acronyms) do
-        -- The definition's name. A Span with an ID so we can create a link.
```

Comment:
Why delete this comment? Is it no longer up-to-date with the code?

Reply:
I think I removed this while modifying this function, but have returned here: 44b114981e2aa9d4037ddfeffd6cda2a088ef37c